cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -725,6 +724,11 @@
   public static final Id.Namespace SYSTEM_NAMESPACE_ID = Id.Namespace.from(SYSTEM_NAMESPACE);
 
   /**
+   * 'cdap' reserved namespace name
```

Comment:
No, we're actually reserving the namespace name 'cdap'. Just in case we need it in future too.